### DISCUSSION TITLE
Build psycopg2 from source, easy way!

### DISCUSSION THREAD

**Discussion by psyhomb** [psycopg/psycopg2#1277] [General]
I have created simple build script for building a `psycopg2` python module from source inside of Docker container with statically linked `libpq`, it was primarily created for AWS lambda functions, but it can be used for any other project that requires `psycopg2` module.

Just suggestion to be added in the [README.rst](https://github.com/psycopg/psycopg2/blob/master/README.rst#installation) file as additional resource:
https://gitlab.com/psyhomb/aws-psycopg2

Cheers, and keep up the good work!

**Comment by szymic1** [comment]
Nice, it will be nice if you add Python 3.9 and ARM platform (now is available for Lambda)

**Reply by psyhomb** [reply]
Hey @szymic1 thanks for reaching out, I've added python3.9 support, you can now download latest prebuilt psycopg2 module [here](https://gitlab.com/psyhomb/aws-psycopg2#python-39). 🙂
